changelog/149.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+op: store Runners on Products and change Runners interface to return an Op
```

Comment:
Since these changelog files will wind up in release notes, what would you think about us standardizing on sentences for the descriptions? So, something like:

```suggestion
op: Store Runners on Products and change Runners interface to return an Op.
```